timelock-impl/src/test/java/com/palantir/atlasdb/timelock/auth/config/PrivilegesConfigurationTest.java

```diff
@@ -68,6 +75,20 @@ public void canDeserializeClientConfig() throws IOException {
         assertThat(deserializedConfiguration).isEqualTo(expectedConfiguration);
     }
 
+    @Test
+    public void canDeserializeClientConfigWithSuffixes() throws IOException {
+        PrivilegesConfiguration deserializedConfiguration =
```

Comment:
just bundle this test with the test above, they're only two tests because you implemented them in separate prs